tests/tools/kopiarunner/kopia_snapshotter_upgrade_test.go

```diff
@@ -0,0 +1,49 @@
+package kopiarunner
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUpgradeFormatVersion(t *testing.T) {
+	baseDir, err := os.MkdirTemp("", t.Name())
+	require.NoError(t, err)
+
+	defer os.RemoveAll(baseDir)
+
+	repoDir, err := os.MkdirTemp(baseDir, "repo")
+	require.NoError(t, err)
+
+	_, err = os.MkdirTemp(baseDir, "source")
+	require.NoError(t, err)
+
+	ks, err := NewKopiaSnapshotter(repoDir)
+	if errors.Is(err, ErrExeVariableNotSet) {
+		t.Skip("KOPIA_EXE not set, skipping test")
```

Comment:
Shouldn't this fail instead of skipping? It seems that otherwise, this would lead to the test silently failing (not executing) due to an environment misconfiguration and give the impression that it has been running without issues.

Reply:
removed the if block